manual/src/html_processing/scss/manual.scss

```diff
@@ -302,7 +302,12 @@ div.caml-example.toplevel div.caml-input::before{
     color:#888
 }
 span.number{
-    padding-right: 1ex;
+    padding-right: 0.5ex;
+}
+ul li .number {
+    display: inline-block;
```

Comment:
The inline-block is displaying `Chapter` and the number on a separate line in firefox which does not seem ideal.

Reply:
That's only in "Contents" (manual001.html), right? Fixed by making a different class for "Chapter n", they probably shouldn't ever be right-aligned anyway.

Reply:
Yes, this was in `Contents` only. Speaking about this page the three digit numbers in contents (e.g `10.2`) overflows a little the `3ex` fixed width. Maybe it should be increased to `3.5ex`?

Reply:
> The three digit numbers in contents (e.g 10.2) overflows a little the 3ex fixed width. Maybe it should be increased to 3.5ex?

Freeze!  Typesetting police!  The ex is a unit of height, so it's wrong to use it for widths.  The em is more appropriate.

Reply:
In CSS, em is whatever the `font-size` of the element is, which might not be related to the width of "m" (or of anything). The ch unit is supposed to be the width of "0", which looks more fitting.